docs/pages/access-controls/sso/google-workspace.mdx

```diff
@@ -236,12 +287,24 @@ $ tctl sso configure oidc --preset google --id <CLIENT-ID> \
 --google-acc '
 {
   "type": "service_account",
-  ...
+  "project_id": "teleport-project-xxxxxx",
+  "private_key_id": "f06exxxxxxxxxxxxxxxdc18",
+  "private_key": "-----BEGIN PRIVATE KEY-----\n0000000000000w==\n-----END PRIVATE KEY-----\n",
+  "client_email": "gwsso@teleport-project-xxxxxx.iam.gserviceaccount.com",
+  "client_id": "11xxxxxxxxxxxxx89",
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://oauth2.googleapis.com/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/gwsso%40teleport-project-xxxxxx.iam.gserviceaccount.com",
```

Comment:
On the one hand, the full JSON example as provided here helps the reader confirm that they are using the correct file. On the other hand, users who copy this command example directly into their CLI will have a lot more backspacing to do to get rid of it before they paste their example, and thanks to your update above there is already a good example of the file.

This is the reasoning why I went with the current form in my last update. I don't feel strongly enough about this to block the update, but it's my two cents.